docs/pages/repo/docs/reference/configuration.mdx

```diff
@@ -209,7 +209,7 @@ Passing an empty array can be used to tell `turbo` that a task is a side-effect
 
 `type: boolean`
 
-Defaults to `true`. Whether or not to cache the task [`outputs`](#outputs). Setting `cache` to false is useful for daemon or long-running "watch" or development mode tasks you don't want to cache. Setting this option to `false` will ignore all other options.
+Defaults to `true`. Whether or not to cache the task [`outputs`](#outputs). Setting `cache` to false is useful for daemon or long-running "watch" or development mode tasks you don't want to cache. Setting this option to `false` will ignore all other options except for [`dependsOn`](#dependson).
```

Comment:
I think we should just remove this sentence entirely. It's not really a guarantee we want to make so universally.
```suggestion
Defaults to `true`. Whether or not to cache the task [`outputs`](#outputs). Setting `cache` to false is useful for daemon or long-running "watch" or development mode tasks you don't want to cache. 
```